deepchem/dock/pose_generation.py

```diff
@@ -16,7 +16,7 @@
 from deepchem.utils.typing import RDKitMol
 from deepchem.utils.geometry_utils import compute_centroid, compute_protein_range
 from deepchem.utils.rdkit_utils import load_molecule, write_molecule
-from deepchem.utils.vina_utils import load_docked_ligands, write_vina_conf
+from deepchem.utils.vina_utils import load_docked_ligands, write_vina_conf, write_gnina_conf, read_gnina_log
```

Comment:
+1 for @peastman's suggestion to rename this `deepchem.utils.docking_utils`.

On a more general note, we should figure out our policy with API stability on utils files. To be safe, we could make the shift backwards compatible by having wrappers in `vina_utils` that emit deprecation warnings.